app/src/main/java/com/androidavanzado/covid19stadistics/util/DateTextFormat.kt

```diff
@@ -0,0 +1,22 @@
+package com.androidavanzado.covid19stadistics.util
+
+import android.annotation.SuppressLint
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+
+class DateTextFormat(val monthToNameMonth : MonthToNameMonth) {
+
+    @SuppressLint("SimpleDateFormat")
+    operator fun invoke(date: Date): String {
+        val dayFormat: DateFormat = SimpleDateFormat("dd")
```

Comment:
This assignment of SimpleDateFormat should be in a global scope to be created once and not each time when call invoke method

Reply:
done